CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [1.2.0] - 2019-02-16
+## [1.2.0-arduino] - 2019-02-16
```

Comment:
Are we really required to do duplicate tags for the same code? Shouldn't everything work with a single tag now that arduino config is setup?

Reply:
Yes, and no.

Arduino branch will have a different folder structure compatible with Arduino’s library manager.

The Arduino tags should target the Arduino branch; regular tag should target master.

eg
master | 1.2.0
master >> Arduino Script >> arduino branch | 1.2.0-arduino

Using two releases is per @kristjank and discussion here:
https://forum.arduino.cc/index.php?topic=596617.0
> The Library Manager indexer job just works off of Git tags. As long as the tag points to something that is compliant, it's fine. You could have one branch of the repo with the Library Manager compatible structure that you point the Library Manager tags at, and another branch of the repository with your existing structure. You should also be able to create tags that point to the non-compliant branch and the Library Manager indexer should just ignore those.

The alternative would be a completely separate Arduino repo with nearly identical code, just laid out differently than PlatformIO and Desktop builds.

I’m open to more discussion if you feel it’s warranted though.

Reply:
Makes sense now, thanks for the link.